dev.Dockerfile

```diff
@@ -1,6 +1,10 @@
 FROM python:3.10-bookworm
 LABEL description="Mage data management platform"
 ARG PIP=pip3
+ARG OTEL_EXPORTER_OTLP_ENDPOINT
+ARG OTEL_EXPORTER_OTLP_HTTP_ENDPOINT
+ENV OTEL_EXPORTER_OTLP_HTTP_ENDPOINT=${OTEL_EXPORTER_OTLP_HTTP_ENDPOINT}
```

Comment:
You don't need to add these environment variables in the dockerfile.

Reply:
removed